clang/include/clang/Sema/SemaInternal.h

```diff
@@ -75,7 +75,7 @@ getDepthAndIndex(UnexpandedParameterPack UPP) {
   if (const auto *TTP = dyn_cast<const TemplateTypeParmType *>(UPP.first))
     return std::make_pair(TTP->getDepth(), TTP->getIndex());
 
```

Comment:
We should add support for SubstTemplateTypeParmPackType / SubstNonTypeTemplateParmPackExpr here